docs/user-guide/walkthrough/k8s201.md

```diff
@@ -28,16 +28,20 @@ scaling.
 
 In order for the kubectl usage examples to work, make sure you have an examples directory locally, either from [a release](https://github.com/GoogleCloudPlatform/kubernetes/releases) or [the source](https://github.com/GoogleCloudPlatform/kubernetes).
 
-Table of Contents
-- [Labels](#labels)
-- [Replication Controllers](#replication-controllers)
-  - [Replication Controller Management](#replication-controller-management)
-- [Services](#services)
-  - [Service Management](#service-management)
-- [Health Checking](#health-checking)
-  - [Process Health Checking](#process-health-checking)
-  - [Application Health Checking](#application-health-checking)
-- [What's Next?](#whats-next)
+**Table of Contents**
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+- [Kubernetes 201 - Labels, Replication Controllers, Services and Health Checking](#kubernetes-201---labels,-replication-controllers,-services-and-health-checking)
```

Comment:
Commas don't seem to work either, but not a big deal since this is the top of the doc. 

Actual anchor is:
`#kubernetes-201---labels-replication-controllers-services-and-health-checking`

We should strip commas @JanetKuo